python/tvm/topi/x86/injective.py

```diff
@@ -17,20 +17,20 @@
 # pylint: disable=invalid-name
 """x86 declaration and schedules."""
 from tvm import te
+from tvm.topi import tag
 from tvm.tir import IntImm
+from tvm.topi.generic.injective import schedule_injective_from_existing
 from ..utils import is_empty_shape
 
 
-def schedule_injective_from_existing(sch, out):
+def schedule_injective_from_existing_ref(sch, out):
```

Comment:
This change and `from tvm.topi.generic.injective import schedule_injective_from_existing` above don't seem necessary.

Reply:
why is that? `schedule_injective_from_existing` that you import at L22 doesn't seem to be used, and the other change is just renaming.

Reply:
I tried to remove it, test fails with crash: File "/home/sshtin/Dev/tvm/python/tvm/topi/x86/__init__.py", line 30, in <module>
    from .reduction import *
  File "/home/sshtin/Dev/tvm/python/tvm/topi/x86/reduction.py", line 21, in <module>
    from .injective import schedule_injective_from_existing
ImportError: Error importing plugin "tvm.testing.plugin": cannot import name 'schedule_injective_from_existing' from 'tvm.topi.x86.injective' (/home/sshtin/Dev/tvm/python/tvm/topi/x86/in

Reply:
I don't know what that error means, but you shouldn't be making this change anyway. If the old `schedule_injective_from_existing` is used by other places, this change will make a different `schedule_injective_from_existing` (in `tvm.topi.generic.injective`) instead. 

One you cannot use the old `schedule_injective_from_existing` at L150? And since `tvm.topi.generic.injective.schedule_injective_from_existing` is just a one line schedule,
https://github.com/apache/tvm/blob/813136401a11a49d6c15e6013c34dd822a5c4ff6/python/tvm/topi/generic/injective.py#L40

you can directly copy that in your concat schedule, to avoid importing and the name conflict issue.

Reply:
Resolved